docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -145,8 +145,10 @@ that is printed out.
 For the exact list of named groups ``@linter`` recognizes, see the `API
 documentation <https://api.coala.io/en/latest/>`__.
 
-For more info generally on regexes, see `Python re module
-<https://docs.python.org/3/library/re.html>`_.
+.. note::
+
+    | Please refer to `Python3 re module <https://docs.python.org/3/library/re.html>`_ and `Dive into python <http://www.diveintopython3.net/regular-expressions.html>`_'s for information 
```

Comment:
Umm.. You don't need to break the line inside ```actual doc``` - Do it inside ```code``` only, as it makes reading code easier on all devices 😃

Reply:
I'm sorry I don't understand which code.